pkg/flowaggregator/flowaggregator.go

```diff
@@ -399,6 +407,10 @@ func (fa *flowAggregator) Run(stopCh <-chan struct{}, wg *sync.WaitGroup) {
 	defer fa.collectingProcess.Stop()
 	go fa.aggregationProcess.Start()
 	defer fa.aggregationProcess.Stop()
+	if fa.dbExportProcess != nil {
+		go fa.dbExportProcess.Start()
+		defer fa.dbExportProcess.Stop()
+	}
 	go fa.flowRecordExpiryCheck(stopCh)
```

Comment:
should this be run conditionally when `fa.externalFlowCollectorAddr != ""`?

maybe this can be done in a future PR, but it would be good to have some symmetry between the clickhouse and flow collector sink.

Ideally having. shared interface would be great:

```go
type FlowSink interface {
    Run(stopCh <-chan struct{})
    SendSet(set ipfixentities.Set) error
    // other methods?
}

type flowAggregator struct {
    // ...
    sinks []FlowSink
}
```

Reply:
I also think a unified interface of flow collector and Clickhouse may not be necessary right now. Since Flow Aggregator is implemented as IPFIX mediator, exporting progress of other sinks like Clickhouse or Kafka may be very different from IPFIX exporting progress.

Reply:
for the first item, I was asking if this:
```go
go fa.flowRecordExpiryCheck(stopCh)
```
should / could become this:
```go
if fa.externalFlowCollectorAddr != "" {
    go fa.flowRecordExpiryCheck(stopCh)
}
```
?

for the second item, I still think that having a unified interface for sinks would be a good idea, but I won't press the issue now

Reply:
@wsquan171 waiting for an answer for the first item? I don't think `flowRecordExpiryCheck` is used at all for ClickHouse but I could be wrong.

Reply:
@antoninbas it is used within the call to `fa.aggregationProcess.ForAllExpiredFlowRecordsDo(fa.sendFlowKeyRecord)`: https://github.com/wsquan171/antrea/blob/a62b9e368e7f247067bd23dab99d3894561bcae1/pkg/flowaggregator/flowaggregator.go#L442
The expiry check is shared logic between both sinks as this is where active and inactive timeout actually happens and the records are popped from FA's internal map and finalized. 
the keep-alive portion of `flowRecordExpiryCheck` is what only applies to external flow collector, and we added nil checks to bypass them if disabled.

Reply:
thanks for the clarification